insights/specs/sos_archive.py

```diff
@@ -70,10 +70,10 @@ class SosSpecs(Specs):
     mount = simple_file("mount")
     multipath__v4__ll = simple_file("sos_commands/multipath/multipath_-v4_-ll")
     netstat = simple_file("netstat")
-    netstat_agn = simple_file("sos_commands/networking/netstat_-agn")
+    netstat_agn = first_of([simple_file("sos_commands/networking/netstat_-agn"), simple_file("sos_commands/networking/netstat_-W_-agn"), simple_file("sos_commands/networking/netstat_-T_-agn")])
     netstat_i = simple_file("sos_commands/networking/netstat_-i")
     netstat_s = simple_file("sos_commands/networking/netstat_-s")
-    nmcli_dev_show = simple_file("sos_commands/networking/nmcli_dev_show")
+    nmcli_dev_show = first_of([simple_file("sos_commands/networking/nmcli_dev_show"), glob_file("sos_commands/networkmanager/nmcli_dev_show_*")])
```

Comment:
This might not work since the second item is a `glob_file` which need supporting of "multi_output".